### PR TITLE
feature: Automated DAST scanning

### PR DESCRIPTION
**Pull request by drcgjung**
## WHAT

This PR introduces automated (and manual) DAST scanning using ZAP.

## WHY

Manual DAST scanning is a tedious but critical effort. ZAP can be automated to perform these scans regularly and give more immediate feedback. 

## FURTHER NOTES

Please set the following (repository) secrets before activating/merging this PR (by personal communication with the requestor)

image

### CODE REVIEW

File: .github/workflows/kics.yml [hunk 1]
```diff
@@ -32,9 +32,7 @@ on:
 
   workflow_dispatch:
 
-  # Since rules may change should run regularily
```
Comment (drcgjung) [inline .github/workflows/kics.yml:35]: @RoKrish14 is that descheduling a common sense in tx?

**Review by drcgjung — COMMENTED** [review]
I'm fine with it. Commiter should remind the secret creation.

**Review by Martin0815bla — CHANGES_REQUESTED** [review]
- Copyright year not set to 2024 in kics.yml and README.md
- github code check failures in openapi.json (Array Without Maximum Number of Items (v3) Array schema has 'maxItems' set) 5 times

**Comment by drcgjung** [comment]
@RoKrish14 have to close this one as there were secrets in your git history

**Pull request closed without merging**